main.cpp

```diff
@@ -275,296 +300,281 @@ struct Bakery
-5 properties:
-    1) number of windows (int)
-    2) size of living room (float)
-    3) room height (float)
-    4) number of bathrooms (int)
-    5) total floor size (float)
-3 things it can do:
-    1) provide shelter
-    2) provide rest
-    3) get dirty
- */
+void AquaPark::relaxMuscles(bool goToSauna, int swimmingIntensity)
+{
+    goToSauna = false;
+    if (swimmingIntensity > 5)
+    {
+        goToSauna = true;
+        std::cout << "You should use one of the sauna's to relax your muscles!" << std::endl;
+    }
+}
+void AquaPark::haveFun(float aTotalWaterSlidesLength, int numberOfSlides)
+{
+    std::cout << "Total length of our " << numberOfSlides << " slides is: " << aTotalWaterSlidesLength << " which means that it's not possible to NOT have fun!" << std::endl;
+}
+bool AquaPark::learnToSwim(int age, int timeSpent)
+{
+    if (age < 10 && timeSpent > 60)
+        return true;
+    else if (age >= 10 && timeSpent > 120)
+        return true;
+
+    return false;
+}
 
 struct House
 {
-    //number of windows (int)
     int numWindows = 8;
-    //size of living room (float)
     float livingRoomSize = 31.8f;
-    //room height (float)
     float roomHeight = 2.8f;
-    //number of bathrooms (int)
     int numBathRooms = 3;
-    //total floor size (float)
     float totalFloorSize = 155.2f;
 
-    //provide shelter
     void provideShelter();
-    //provide rest
     void provideRest(float sleepQuality);
-    //get dirty
-    bool getDirty(int numPeople, int numAnimals, float totalFloorSize);
+    bool getDirty(int numPeople, int numAnimals, float hTotalFloorSize);
 };
 
- /*
-Thing 5) vca
-5 properties:
-    1) minimum gain (double)
-    2) maximum gain (double)
-    3) initial gain (double)
-    4) modulation attenuator (float)
-    5) inputs (int)
-3 things it can do:
-    1) set inital gain
-    2) add rhythm to signal
-    3) silence the signal
- */
+void House::provideShelter()
+{
+}
+
+void House::provideRest(float sleepQuality)
+{
+    if (sleepQuality > 3)
+        std::cout << "Conratulations, you managed to get rest!" << std::endl;
+    else
+        std::cout << "Sorry buddy, you'll be sleepy for the rest of the day." << std::endl;
+}
+
+bool House::getDirty(int numPeople, int numAnimals, float hTotalFloorSize)
+{
+    if (hTotalFloorSize / (numPeople+numAnimals) > 20)
+        return true;
+
+    return false;
+}
 
 struct VCA
 {
-    //minimum gain (double)
     double minGain = -100.66;
-    //maximum gain (double)
     double maxGain = 10.5;
-    //initial gain (double)
     double initGain = -20.1;
-    //modulation attenuator (float)
     float modAtt = -56.2f;
-    //inputs (int)
     int inputs = 2;
 
-    //set inital gain
-    float setInitGain(double minGain, double maxGain);
-    //add rhythm to signal
-    void modulateOutput(float modAtt, float modSpeed);
-    //silence the signal
+    double setInitGain(double vMinGain, double vMaxGain);
+    void modulateOutput(float vModAtt, float modSpeed);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise